.azure-ci.yml

```diff
@@ -0,0 +1,114 @@
+variables:
+  - name: _TeamName
```

Comment:
I think an array is not necessary here. Any particular reason why you followed this notation?